dask_kubernetes/operator/core.py

```diff
@@ -0,0 +1,170 @@
+import subprocess
+import json
+import re
+import time
+
+from distributed.core import rpc
+from distributed.deploy import Cluster
+
+from distributed.utils import Log, Logs
+
+from dask_kubernetes.auth import ClusterAuth
+from .daskcluster import (
+    build_cluster_spec,
+    build_worker_group_spec,
+)
+
+
+class KubeCluster2(Cluster):
+    """Launch a Dask Cluster on Kubernetes using the Operator"""
+
+    def __init__(
+        self,
+        name,
+        namespace="default",
+        image="daskdev/dask:latest",
+        replicas=3,
+        resources={},
+        env={},
+        asynchronous=False,
+        auth=ClusterAuth.DEFAULT,
+        **kwargs,
+    ):
+        self.name = name
+        self.namespace = namespace
+        self.core_api = None
+        self.custom_api = None
+        self.image = image
+        self.replicas = replicas
+        self.resources = resources
+        self.env = env
+        self.auth = auth
+
+        super().__init__(asynchronous=asynchronous, **kwargs)
+        if not self.asynchronous:
+            self._loop_runner.start()
+            self.sync(self._start)
+
+    async def _start(self):
+        data = build_cluster_spec(
+            self.name, self.image, self.replicas, self.resources, self.env
+        )
+        with open("data.json", "w") as jfile:
+            json.dump(data, jfile)
+        cluster = subprocess.check_output(
+            [
+                "kubectl",
+                "apply",
+                "-f",
+                "data.json",
+                "-n",
+                self.namespace,
+            ],
+            encoding="utf-8",
+        )
+        time.sleep(5)
```

Comment:
Waits for the pods to be in Running state. Not ideal, but I could not get it to work with `while "Running" not in kubectl ...`

Reply:
I don't feel confident merging with a sleep in here like this. I know we are going to fix some things up later but this is probably going to be very brittle. 

Could we use `kubectl wait` here instead? 

https://stackoverflow.com/questions/53536907/kubectl-wait-for-condition-complete-timeout-30s